### ISSUE TITLE
Tiles

### ISSUE THREAD

**Issue by NickMcConnell** [NickMcConnell/FAangband#90]
Leave until monsters are done

**Comment by davidk64fnq** [comment]
> Leave until monsters are done

Does this mean there are plans to revise the monsters encountered in FAangband at some point? For example I encountered creeping coins just now that doesn't feel very Tolkien First Age! Out of curiosity is the maddened weasel canon :)

**Comment by NickMcConnell** [comment]
No, that was written before I had modified the monsters from V.  I'm not too fussed about things like creeping coins (I mean, Tolkien doesn't say there **weren't** any); I'm more concerned about removing stuff that clashes (like the old dark elves), and things that are definitely not First Age.  Maddened weasel I think is [entirely reasonable](https://books.google.com.au/books?id=FN4Zy2tqupEC&pg=PT279&lpg=PT279&dq=%22great+weasels+full+of+the+thirst+of+blood%22&source=bl&ots=-WRIexttdd&sig=ACfU3U16g0nzatnun0soA13iUA3-m-KFYw&hl=en&sa=X&ved=2ahUKEwikiPqa1_j0AhWGS2wGHa7aBHcQ6AF6BAgmEAM#v=onepage&q=%22great%20weasels%20full%20of%20the%20thirst%20of%20blood%22&f=false).

**Comment by davidk64fnq** [comment]
"... great weasels full of the first of blood..."

I'll see them in a different light next encounter!
